src/ORM/Association/BelongsToMany.php

```diff
@@ -673,8 +673,8 @@ protected function _saveLinks(EntityInterface $sourceEntity, $targetEntities, $o
     /**
      * Associates the source entity to each of the target entities provided by
      * creating links in the junction table. Both the source entity and each of
-     * the target entities are assumed to be already persisted, if the are marked
-     * as new or their status is unknown, an exception will be thrown.
+     * the target entities are assumed to be already persisted, if they are marked
+     * as new or their status is unknown then an exception will be thrown.
```

Comment:
See my last part: Making it a new sentence makes the whole two sentences way easier to understand :)
But sure, out of scope of your fix.